src/main/java/com/maddyhome/idea/vim/group/IjOptionProperties.kt

```diff
@@ -29,6 +29,7 @@ public open class GlobalIjOptions(scope: OptionAccessScope) : OptionsPropertiesB
   public val lookupkeys: StringListOptionValue by optionProperty(IjOptions.lookupkeys)
   public var trackactionids: Boolean by optionProperty(IjOptions.trackactionids)
   public var visualdelay: Int by optionProperty(IjOptions.visualdelay)
+  public var showmodewidget: Boolean by optionProperty(IjOptions.showmodewidget)
```

Comment:
Is this a final name for this option? Are we going to keep the option if we decide to have this widget?

Reply:
Nope, this widget is a successor of the showmode widget that we used to have in status bar (which is not visible with the new UI)
I plan to collect feedback and replace the old widget with a new one, so it's a temporary option. In future the `showmode` option will be used.